chia/wallet/nft_wallet/nft_puzzles.py

```diff
@@ -83,21 +83,29 @@ def get_nft_info_from_puzzle(puzzle: Program, nft_coin: Coin) -> NFTInfo:
     data_uris = []
     for uri in uncurried_nft.data_uris.as_python():
         data_uris.append(str(uri, "utf-8"))
+    meta_uris = []
```

Comment:
we should use type hinting here:
meta_uris: List[str] = []

Reply:
good catch